.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+## Pull Request Title
+
+> Explain  in a few lines the purpose of this pull request
+
+**Issue link:** *required*
+
+**Doc Issue:** *Optional, link issue from [documentation repository](https://github.com/wso2/docs-ob/issues)*
+
+**Applicable Labels:** *Spec, product, version, type (specify requested labels)*
+
+------
+
+### Development Checklist
+
+1. [ ] Build complete solution with pull request in place.
+2. [ ] Ran checkstyle plugin with pull request in place.
+3. [ ] Ran Findbugs plugin with pull request in place.
+4. [ ] Ran FindSecurityBugs plugin and verified report.
+5. [ ] Formatted code according to WSO2 code style.
+6. [ ] Have you verify the PR does't commit any keys, passwords, tokens, usernames, or other secrets?
+7. [ ] Migration scripts written (if applicable).
+8. [ ] Have you followed secure coding standards in [WSO2 Secure Engineering Guidelines](http://wso2.com/technical-reports/wso2-secure-engineering-guidelines)?
+
+### Testing Checklist
+
+1. [ ] Written unit tests.
+2. [ ] Documented test scenarios(link available in guides).
+3. [ ] Written automation tests (link available in guides).
+4. [ ] Verified tests in multiple database environments (if applicable).
+5. [ ] Verified tests in multiple deployed specifications (if applicable).
+6. [ ] Tested with OBBI enabled  (if applicable).
+7. [ ] Tested with specification regulatory conformance suites  (if applicable).
+
+**Automation Test Details**
```

Comment:
Since the test repo is closed source, do we need this test suite details

Reply:
fixed with https://github.com/wso2/financial-open-banking/pull/1/commits/cb8540774ef85e4e7511b278795697650c0a44a8